src/components/app/details/cicdHistory/Sidebar.tsx

```diff
@@ -32,6 +32,9 @@ const Sidebar = React.memo(({ type, filterOptions, triggerHistory, hasMore, setP
         if (type === HistoryComponentType.CI) {
             setPagination({ offset: 0, size: 20 })
             push(generatePath(path, { appId, pipelineId: selectedFilter.value }))
+        } else if (type === HistoryComponentType.GROUP_CI || type === HistoryComponentType.GROUP_CD){
+            setPagination({ offset: 0, size: 20 })
```

Comment:
We should extract out the common part and change the payload based on condition and pass